src/redteam/extraction/entities.ts

```diff
@@ -12,7 +44,7 @@ export async function extractEntities(provider: ApiProvider, prompts: string[]):
     Each line in your response must begin with the string "Entity:".
   `;
 
-  return callExtraction(provider, prompt, (output: string) => {
+  return await callExtraction(provider, prompt, (output: string) => {
```

Comment:
I think there is debate about this but I usually just `return` in these situations
```suggestion
  return callExtraction(provider, prompt, (output: string) => {
```

Reply:
This was intentional. I did it here because of the try-catch (which also may be unnecessary). If you dislike the pattern I can write it to a variable first.

Reply:
It was not intentional! I apologize, thanks for the callout.